secure-embedded-cache/pom.xml

```diff
@@ -65,9 +65,25 @@
       </dependency>
       <dependency>
          <groupId>org.jboss.resteasy</groupId>
-         <artifactId>jaxrs-api</artifactId>
-         <version>3.0.11.Final</version>
+         <artifactId>resteasy-client</artifactId>
+         <version>3.0.24.Final</version>
+         <scope>test</scope>
+      </dependency>
+      <dependency>
+         <groupId>org.jboss.resteasy</groupId>
+         <artifactId>resteasy-jaxrs</artifactId>
+         <version>3.0.24.Final</version>
          <scope>provided</scope>
+         <exclusions>
+            <exclusion>
```

Comment:
How about using JEE API jar, like this one: https://mvnrepository.com/artifact/org.jboss.spec/jboss-javaee-all-7.0 ?

Reply:
Looking at the artifact's pom, it has only dependencies on jboss-servlet-api_3.1_spec, jboss-jsp-api_2.3_spec and it shades them inside. Which dependencies you think this new dep should replace? Thanks:)

Reply:
oooh crap! My bad!